src/api/feed-discovery/src/util.js

```diff
@@ -20,6 +20,27 @@ const toTwitchFeedUrl = (twitchChannelUrl) => {
   throw new Error('not a Twitch URL');
 };
 
+const isFeedUrl = async (url) => {
+  try {
+    const { statusCode, headers } = await got(url);
+    const contentType = headers['content-type'];
+    const validContentTypes = [
+      'application/xml',
+      'application/rss+xml',
+      'application/atom+xml',
+      'application/x.atom+xml',
+      'application/x-atom+xml',
+      'application/json',
```

Comment:
I'm not sure that the `rss-parser` module we use (see https://github.com/rbren/rss-parser) is capable of parsing all of these types.  We might want to limit it to valid Atom + RSS only.